src/frame/worker/crypto/recrypt/RecryptWasm.ts

```diff
@@ -248,6 +248,16 @@ export const createRequestSignature = (
     };
 };
 
+/**
+ * Generate a schnorr signature using the group's private key over the group's provided id
+ * @param groupPrivateKey
+ * @param groupPublicKey
+ * @param groupId
+ */
+export const generateAddMemberSignature = (groupPrivateKey: Uint8Array, groupPublicKey: PublicKey<string>, groupId: string) => {
```

Comment:
Won't this same signature calculation be used when we add admins to the group too? Should this be renamed to `calculateGroupModificationSignature` or similar instead?

Reply:
Seems like this is pretty general purpose and could maybe be called `signString` - take a private key, a public key, and a string.

Reply:
I went with schnorrSignString.